README.md

```diff
@@ -588,7 +580,7 @@ Snapshot Debugger.
 | Argument                      | Description |
 |-------------------------------|-------------|
 | `-h`, `--help`                | Show this help message and exit. |
-| `--database-url DATABASE_URL` | Specify the database URL for the CLI to use. This should only be used as an override to make the CLI talk to a specific instance and isn't expected to be needed. If you are on the Spark plan and want the CLI to use the default instance use the `--use-default-rtdb` flag instead. If neither of the `--database-id` or `--use-default-rtdb` flags are used with the `init` command, the CLI uses the default url. |
+| `--database-url DATABASE_URL` | Specify the database URL for the CLI to use. This should only be used as an override to make the CLI talk to a specific instance and isn't expected to be needed. If you are on the Spark plan and want the CLI to use the default instance use the --use-default-rtdb flag instead. If neither of the --database-id or --use-default-rtdb flags are used with the init command, the CLI uses the default url.  This value may be specified either via this command line argument or via the `SNAPSHOT_DEBUGGER_DATABASE_URL` environment variable.  When both are specified, the value from the command line takes precedence. |
```

Comment:
add the `code font` back to --database-id and --use-default-rtdb flags in the description.

Reply:
Thanks, this has been addressed in a recent commit that modified the database-url description.